src/components/typography/Typography.css

```diff
@@ -0,0 +1,129 @@
+h1, h2, h3, h4, h5, h6, .bodyLarge, .bodyMedium, .bodySmall, .cardTitle, .cardBody, .cardTags {
+  font-family: 'DM Sans', sans-serif;
+}
```

Comment:
Think we need to import DM Sans or something here for it to apply, maybe check how main website does it